assembl/graphql/view.py

```diff
@@ -13,18 +15,40 @@
 class LoggingMiddleware(object):
     def resolve(self, next, source, gargs, context, info, *args, **kwargs):
         if source is None:
-            variables = {}
+            modified_variables = {}
             for key, value in info.variable_values.items():
                 if 'password' in key or 'Password' in key:
-                    variables[key] = 'xxxxxxxxxxxxx'
+                    modified_variables[key] = 'xxxxxxxxxxxxx'
                 else:
-                    variables[key] = value
+                    modified_variables[key] = value
+
             getLogger().debug(
                 'graphql', op=info.operation.operation,
-                opname=info.operation.name.value, vars=variables)
+                opname=info.operation.name.value, vars=modified_variables)
         return next(source, gargs, context, info, *args, **kwargs)
 
 
+def get_graphql_params(request, data):
+    query, variables, operation_name = original_get_graphql_params(request, data)
+    modified_variables = {}
+    for key, value in variables.items():
+        if 'password' in key or 'Password' in key:
+            modified_variables[key] = 'xxxxxxxxxxxxx'
+        else:
+            modified_variables[key] = value
+
+    operation = query.split()[0]
+    getLogger().debug(
```

Comment:
Do we lose logging under non-API call's then? Eg. no logs for testing, etc? Not a big deal otherwise.

Reply:
I don't understand what you mean. The logging is the same as before my changes, we log in debug every graphql request.

Reply:
no, this is not the same code path. Using a middleware means calling the `execute` will also trigger the logging. However, in tests, when we call `schema.execute`, we miss this logging.

Reply:
Significant? Probably not. But, something to be aware of.

Reply:
This is graphql_wsgi which calls schema.execute with given middleware. In tests we use schema.execute directly without middleware, so no logs in tests.  `get_graphql_params` here is used in graphql_wsgi just before the `schema.execute` call. So there is no changes in logging.